app/code/community/Sailthru/Email/Model/Client/Content.php

```diff
@@ -110,19 +111,13 @@ public function getProductData(Mage_Catalog_Model_Product $product)
                 'isAvailable'  => $product->isAvailable(),
                 'isVirtual'  => $product->isVirtual(),
                 'isRecurring' => $product->isRecurring(),
-                'isInStock'  => $product->isInStock(),
+                'isInStock'  => $stockItem->isInStock(),
                 'weight'  => $product->getWeight()
             )
         );
 
         if ($isSimple) {
-            if ($product->getStockItem()->getQty() || $product->getQty()) {
-                $inv = $product->getStockItem()->getQty() ?: intval($product->getQty());
-                if ($inv < 0) {
-                    $inv = 0;
-                }
-                $data['inventory'] = $inv;
-            }
+                $data['inventory'] = $stockItem->getStockQty(); // This function works even for composite products.
```

Comment:
What does this return when a product doesn't have inventory?

Reply:
This seems concerning because it could accidentally filter out items that don't directly store their inventory.